frontend/src/global_styles/content/_enterprise.sass

```diff
@@ -1,34 +1,31 @@
 @import '../../app/spot/styles/sass/variables'
 
 .upsale-notification
-  max-width: 1000px
+  max-width: 50vw
   margin: auto
   padding-top: 20px
   text-align: center
 
 
   .widget-box--teaser-image
-    width: 100%
-    height: 40vh
+    width: 90%
+    height: auto
     margin-bottom: 20px
-    border: 1px solid $spot-color-main-dark
-    border-radius: 20px
-    padding: 20px
+    box-shadow: 4px 4px 10px rgb(0 0 0 / 15%)
+    border-radius: 25px
   
   .widget-box--teaser-image_default
     width: 30%
     margin-bottom: 20px
-    border: 1px solid $spot-color-main-dark
-    border-radius: 20px
-    padding: 20px
+    box-shadow: 4px 4px 10px rgb(0 0 0 / 15%)
+    border-radius: 25px
     
   .widget-box--teaser-video
-    width: 100%
```

Comment:
Since the image and the video share exactly the same styles, they can be combined.